src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -155,6 +155,21 @@ export class DynamicFacet extends Component implements IDynamicFacet {
       section: 'Sorting'
     }),
 
+    /**
+     * Specifies a custom order by which to sort the dynamic facet values.
+     *
+     * **Example:**
+     *
+     * You could use this option to specify a logical order for support tickets, such as:
+     * ```html
+     * <div class="CoveoDynamicFacet" data-field="@ticketstatus" data-title="Ticket Status" data-tab="All" data-custom-sort="New,Opened,Feedback,Resolved"></div>
+     * ```
+     *
+     * **Note:**
+     * > The [`DynamicFacetRange`]{@link DynamicFacetRange} component does not support this option.
+     */
```

Comment:
Source: [`IFacetOptions.customSort`](https://github.com/coveo/search-ui/blob/master/src/ui/Facet/Facet.ts#L302)

Reply:
An important note to add is that for the option to work correctly, the number of values to display in the facet should match the number of facet values.

For example, if there are 20 values in the index, but the facet is only displaying 5 and the `customSort` value is one of the 15 not returned, the feature will not work as expected. To guarantee that it works, the number of displayed values would need to be set to 20.